processor/deltatocumulativeprocessor/internal/streams/errors.go

```diff
@@ -0,0 +1,18 @@
+package streams
+
+import (
+	"fmt"
+)
+
+func Error(id Ident, err error) error {
+	return StreamErr{Ident: id, Err: err}
+}
+
+type StreamErr struct {
```

Comment:
Is this better unexported, since clients of this package should probably use the `Error` function?

```suggestion
type streamErr struct {
```

Reply:
it's exported so it can be used for `errors.As`